$NetBSD: patch-ab,v 1.1.1.1 2012/05/08 22:39:39 pettai Exp $

Look for the configuration files in SYSCONFDIR instead of share

--- ./lib/DNSCheck/Config.pm.orig	2009-04-04 13:49:09.000000000 +0200
+++ ./lib/DNSCheck/Config.pm	2009-04-04 13:49:43.000000000 +0200
@@ -52,7 +52,7 @@
 
     my %arg = @_;
 
-    $self->{configdir} = catfile($Config{'siteprefix'}, 'share/dnscheck');
+    $self->{configdir} = catfile($Config{'siteprefix'}, '@SYSCONFDIR@/dnscheck');
     $self->{configdir} = $arg{'configdir'} if defined($arg{'configdir'});
 
     $self->{sitedir} = $self->{configdir};
